site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -1976,43 +1954,113 @@ else if (state.getAttribute(STATE_CM_REQUESTED_SECTIONS) != null) {
 			context.put("fromArchive", state.getAttribute(STATE_UPLOADED_ARCHIVE_NAME));
 
 			return (String) getContext(data).get("template") + TEMPLATE[10];
+		case STATE_TEMPLATE_INDEX_MANAGE_PARTICIPANTS_INT:
+			/*
+			 * buildContextForTemplate chef_site-siteInfo-manageParticipants.vm
+			 */
+
+			// Put the link for downloading participant into the context
```

Comment:
Printing instead of downloading?

Reply:
It's for downloading the PDF of the participants list. I'll update the comment and the method name (it's had this name historically, but it's misleading).